test/test-classTetrisMoves.js

```diff
@@ -279,15 +280,15 @@ describe("All moves tetromino", function () {
         assert.deepEqual(gameTetris.tetromino.coordinates, expectedMovesRightTicksO)
         stubForGetRandomTypeOfTetromino.returns("I")
         gameTetris.tick()
-        let expectedFieldAddOSecond = tetris.buildField(`
+        let expectedFieldAddIThird = tetris.buildField(`
             --------
             -----X--
             -----X--
             ---XXX--
             --XXXX--
             XXXX-XXX
         `)
-        assert.deepEqual(gameTetris.field.field, expectedFieldAddOSecond)
+        assert.deepEqual(gameTetris.field.field, expectedFieldAddIThird)
         gameTetris.tick()
         let expectedFieldAddOThird = tetris.buildField(`
```

Comment:
you didn't change all other variable names - this has nothing to do with O-tetromino, for example